src/components/dialogs/dialogUtils.js

```diff
@@ -121,6 +121,10 @@ export const percentageTextField = {
 };
 export const func_identity = (e) => e;
 
+// TODO replace with roundToDefaultPrecision(e * 1e6) and roundToDefaultPrecision(e / 1e6) cf PR 1146
+export const unitToMicroUnit = (e) => e * Math.pow(10, 6);
```

Comment:
`e * 1e6` is more concise and clear than `e * Math.pow(10, 6)`

Reply:
yes, will be change very soon by upcoming "roundTo" function

Reply:
changed after merge